scripts/gathering/generateLPRewards.ts

```diff
@@ -0,0 +1,38 @@
+import fs from 'fs';
+import {TheGraph} from '../utils/thegraph';
+import {getBlockArgs} from '../utils/utils';
+
+const blockNumber = getBlockArgs(0);
+
+interface Staker {
+  id: string;
+}
+
+const theGraph = new TheGraph(
+  'https://api.thegraph.com/subgraphs/name/nicovrg/liquidity-mining'
```

Comment:
Is it the same src code as in https://github.com/thesandboxgame/staking-graph ?

Reply:
yes it is. i had to redeploy the subgraph because it was not indexed anymore.
i've looked for the cause of the de-indexing but i only have unverified hypothesis so ...  

you'll find the details here if you want to check the little doc i made
https://docs.google.com/document/d/1Ga4akc0e2d1ab-OKM0KYDngO442BCR0Sl6KVE017Cvs/edit

Reply:
Why not reuse the one https://github.com/thesandboxgame/staking-graph as Leon said ?
We could update it with the new subgraph

Reply:
As Nicolas is leaving, let's merge it and we'll do something about it later.